src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1702,11 +1714,8 @@ namespace dxvk {
       ImGui::SetCursorPosY(y + (thumbnailSize - extent.y) / 2.f);
 
       if (ImGui::ImageButton(texImgui.texID, extent)) {
-        if (isListFiltered) {
-          toggleTextureSelection(texHash, uniqueId, listRtxOption.textureSetOption->getValue());
-        } else {
-          clickedOnTextureButton = true;
-        }
+        // removed "toggleTextureSelection" logic as it was not working as intended even before
```

Comment:
Maybe just remove this comment, so it would not be confusing in the future what was here before?

Reply:
I've restored the original logic here as I've fixed the root issue and removed the comment